app/src/main/java/com/richpathanimator/sample/AnimationSamplesActivity.java

```diff
@@ -214,7 +214,7 @@ public void animateLoveFace(View view) {
                 .animate(rEye, lEye)
                 .interpolator(new LinearInterpolator())
                 .duration(800)
-                .repeatMode(RichPathAnimator.RESTART)
+//                .repeatMode(RichPathAnimator.RESTART)
```

Comment:
why we commented it? it doesn't work?

Reply:
I'm not sure, but I guess it's a mistake.
Let's back to the our conversations #53 (comment).
I found something is wrong, then you fixed repeatMode and repeatCount bugs.

I think we should modify this line to uncomment.